ifood/buck
# Upgrade JUnixSocket to 2.4.0 for Apple Silicon support

Upgrade `junixsocket` to version `2.4.0` to fix watchman integration in Apple Sillicon.

![image](https://user-images.githubusercontent.com/50594398/183715892-32342b7c-cfc7-43b8-8483-ad0a8adee189.png)

**HOW TO TEST:** 

Just build the same buck target twice, the second command should run instantly and without watchman warnings.

**Note: We can drop this commit before rebasing because `facebook/dev` already have this fix.**
